rundeckapp/grails-app/assets/javascripts/executionControl.js

```diff
@@ -473,107 +473,139 @@ var FollowControl = function (eid, elem, params) {
         var rowcount= this.countTableRows(this.cmdoutputtbl);
         var compacted = data.compacted;
         var compactedAttr = data.compactedAttr;
+
+        /**Called after table update is finished for housekeeping and fetching next batch if necessary **/
+        var doFinishedLogUpdate = function() {
```

Comment:
I wouldn't mind a bit more in the way of commenting on the what this code does. If you think we'll most likely replace this with that Vue list/table component you were showing me the other day, please feel free to disregard this.